### ISSUE TITLE
Set up our own bucket for GCS integration tests

### ISSUE THREAD

**Issue by mpenkov** [RaRe-Technologies/smart_open#450]
Related to above, does it make sense long term for me to be the one running those tests? Would RaRe-Technologies be willing to create a Google Cloud account to run the tests on? I would be happy to help in any way with that, but I don't think it makes sense to have the account linked to me. I believe you get $300 in credits for the first year and billing information isn't even required during that time. I can't imagine these tests costing much money to run after that. I'm happy to open a new issue to discuss this further if that is more appropriate. (cc: @piskvorky)

_Originally posted by @petedannemann in https://github.com/RaRe-Technologies/smart_open/pull/448#issuecomment-605658630_

**Comment by piskvorky** [comment]
@petedannemann we have bad experience with free time-limited trial accounts, because when they expire it's chaos.

Unfortunately we have zero budget for open source, this is all 100% voluntary work (which we sink a lot of our time into).

**Comment by petedannemann** [comment]
@piskvorky if I did a cost estimate would you accept a donation?

**Comment by mpenkov** [comment]
@piskvorky The price is comparable to what we're paying for our S3 bucket (https://cloud.google.com/storage/pricing#price-tables). We don't move large amounts of data around, so it'll be no more than a few cents per month, from what I understand.

**Comment by ddelange** [comment]
no longer relevant after #599
